### DISCUSSION TITLE
Support for Network Card ISA DP83905AVQB ATLANTIC or DP83905 chip

### DISCUSSION THREAD

**Discussion by FigueiredoStable** [86Box/86Box#3881] [General]
I have a proprietary system and it only accepts this network drive, in my research I discovered these specifications:

Here are some technical specifications of the ISA DP83905AVQB ATLANTIC network card:

![Photo 295 (1)](https://github.com/86Box/86Box/assets/88445452/9feaee89-18e4-4a4a-8210-cd8a2051a959)

Network chip: DP83905
Speed: 10 Mbps
Protocol: Ethernet
Interface: ISA

The DP83905AVQB ATLANTIC was an important piece of computer hardware history and helped popularize local area networks on personal computers. She is still fondly remembered by many computer enthusiasts.

Additional specifications:

Support for IRQ 3, 4, 5, 7, 9, 10, 11, 12, 14, 15
Support for DMA 1, 3, 5, 6
Support for RAM memory from 1 MB to 8 MB
Dimensions: 120 x 96 x 1.6 mm

The DP83905 chipset was used in a variety of ISA network cards from the 1990s. Some of the more popular network cards that used this chipset include:

Atlantic DP83905AVQB

Atheros LAN9000

D-Link DFE-500TX

Intel PRO/100

Is there any way to support this chipset?

**Comment by TC1995** [comment]
by technical details I meant a full register/port datasheet.

**Reply by FigueiredoStable** [reply]
I find this datasheet 
[DP83905 AT LANTIC.pdf](https://github.com/86Box/86Box/files/13654795/DP83905.AT.LANTIC.pdf)
